pkg/serving/config_changes.go

```diff
@@ -331,14 +331,26 @@ func UpdateContainerArg(template *servingv1.RevisionTemplateSpec, arg []string)
 	return nil
 }
 
-// UpdateContainerPort updates container with a give port
-func UpdateContainerPort(template *servingv1.RevisionTemplateSpec, port int32) error {
+// UpdateContainerPort updates container with a given name:port
+func UpdateContainerPort(template *servingv1.RevisionTemplateSpec, port string) error {
 	container, err := ContainerOfRevisionTemplate(template)
 	if err != nil {
 		return err
 	}
+	var containerPort int
```

Comment:
int32?

Reply:
Thanks for all the feedback @navidshaikh . Had to use int64 because strconv.ParseInt returns int64, later casted to int32.